### ISSUE TITLE
False positive on Psalter safe-types with round()

### ISSUE THREAD

**Issue by orklah** [vimeo/psalm#3092]
Hi,

For the following code:

```php
<?php
declare(strict_types=1);

function test(){
    return round(12);
}
```

And the following command:
`vendor\bin\psalm --alter --issues=MissingReturnType --safe-types`

Psalter will transform the code into:

```php
<?php
declare(strict_types=1);

function test(): int{
    return round(12);
}
```

However, round() actually returns a float. This will cause a Type Error:
https://3v4l.org/MRD6H

Thanks

**Comment by muglug** [comment]
For whatever dumb reason, I thought `round` sometimes returned an int. But it never does
